### PR TITLE
Lp/fix/524/simplify studio parameter for sorting

### PR DESCRIPTION
**Pull request by HegelPro**

### CODE REVIEW

File: packages/codegen/src/generators/react-typescript/common/template/Cards.tsx.ejs [hunk 1]
```diff
@@ -257,7 +257,7 @@ export function <%= componentName %>() {
             />
           </Card>
         <% } -%>
-        <% if (mode === 'edit') { -%>
+        <% if (mode === 'edit' || withOrderBy) { -%>
```
Comment (vadimbasko) [inline packages/codegen/src/generators/react-typescript/common/template/Cards.tsx.ejs:260]: @HegelPro please add screens in example-app to check this case

File: packages/codegen/src/generators/react-typescript/common/template/List.tsx.ejs [hunk 2]
```diff
@@ -257,7 +257,7 @@ export function <%= componentName %>() {
             />
           </Card>
         <% } -%>
-        <% if (mode === 'edit') { -%>
+        <% if (mode === 'edit' || withOrderBy) { -%>
```
Comment (vadimbasko) [inline packages/codegen/src/generators/react-typescript/common/template/List.tsx.ejs:260]: @HegelPro please add screens in example-app to check this case - `mode != edit && withOrderBy`. I think we need two example with `mode == view` and `mode == view with details`

File: packages/codegen/src/generators/react-typescript/common/template/Table.tsx.ejs [hunk 3]
```diff
@@ -298,7 +295,7 @@ export function <%= componentName %>() {
             />
           </Card>
         <% } -%>
-        <% if (mode === 'edit') { -%>
+        <% if (mode === 'edit' || withOrderBy) { -%>
```
Comment (vadimbasko) [inline packages/codegen/src/generators/react-typescript/common/template/Table.tsx.ejs:298]: @HegelPro please add screens in example-app to check this case - `mode != edit && withOrderBy`. I think we need two example with `mode == view` and `mode == view with details`

**Review by vyacheslav-pushkin — APPROVED** [review]
Please address @vadimbasko 's comment and merge.

File: packages/codegen/src/generators/react-typescript/common/template/Table.tsx.ejs (hunk 3 above)
Comment (vadimbasko) [reply packages/codegen/src/generators/react-typescript/common/template/Table.tsx.ejs:298]: Also, it looks like for `mode == edit && !withOrderBy` empty `<ButtonPanel/>`  will be generated. I think it is not correct. Please check for other collection types too (cards and list).

**Pull request merged**